src/api/app/controllers/webui/package_controller.rb

```diff
@@ -781,8 +781,22 @@ def set_job_status
   def live_build_log
     required_parameters :arch, :repository
 
-    @project = Project.get_by_name(params[:project])
-    @package = Package.get_by_project_and_name(params[:project], params[:package], use_source: false, follow_project_links: true)
+    if Project.exists_by_name(params[:project])
```

Comment:
Why are we not using the general before filters (`set_project`, `require_package`) here?